pennylane/ops/qubit.py

```diff
@@ -1048,6 +1048,14 @@ class CRX(Operation):
     num_wires = 2
     par_domain = "R"
     grad_method = "A"
+
+    # gradient recipe
+    c1 = (np.sqrt(2) - 4 * np.cos(np.pi / 8)) / (4 - 8 * np.cos(np.pi / 8))
+    c2 = (np.sqrt(2) - 1) / (4 * np.cos(np.pi / 8) - 2)
+    a = np.pi / 2
+    b = 3 * np.pi / 4
+    grad_recipe = ([[c1, 1, a], [-c1, 1, -a], [-c2, 1, b], [c2, 1, -b]],)
```

Comment:
Is this the same for each one? Worth pulling it into one spot for maintainability?

Reply:
done!